src/main/steps/applicant2/upload-your-documents/template.njk

```diff
@@ -0,0 +1,131 @@
+{% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
+{% from "govuk/components/details/macro.njk" import govukDetails %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/file-upload/macro.njk" import govukFileUpload %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% extends "common/page.njk" %}
+
+{% block page_content %}
+  {{ govukErrorSummary({
+    titleText: errorSummaryHeading,
+    classes: "hidden",
+    attributes: {
+      id: "uploadErrorSummary"
+    },
+    errorList: [
+      {
+        text: errors.applicant2UploadedFiles.errorUploading,
+        href: "#upload",
+        attributes: {
+          id: "errorGeneric",
+          class: "hidden"
+        }
+      },
+      {
+        text: errors.applicant2UploadedFiles.fileSizeTooBig,
+        href: "#upload",
+        attributes: {
+          id: "errorFileSizeTooBig",
+          class: "hidden"
+        }
+      },
+      {
+        text: errors.applicant2UploadedFiles.fileWrongFormat,
+        href: "#upload",
+        attributes: {
+          id: "errorFileWrongFormat",
+          class: "hidden"
+        }
+      }
+    ]
+  }) }}
+
+  <h1 class="govuk-heading-l">{{ title }}</h1>
+  <p class="govuk-body">{{ youNeed }}</p>
+  <ul class="govuk-list govuk-list--bullet">
+    <li>{{ certificate if formState.inTheUk === 'YES' else certificateForeign }}</li>
+    {% if formState.certifiedTranslation === 'YES' %}<li>{{ certificateForeignTranslation }}</li>{% endif %}
+    {% if formState.applicant2LastNameChangedWhenRelationshipFormed === 'YES' or
+          formState.applicant2NameChangedSinceRelationshipFormed === 'YES'%}
+      <li>{{ proofOfNameChange }}</li>
+    {% endif %}
+  </ul>
+  {{ govukWarningText({
+    text: warningPhoto,
+    iconFallbackText: "Warning"
+  }) }}
+  {% set infoTakePhotoDetails %}
+    <ol class="govuk-list govuk-list--number">
+      <li>{{ infoBullet1 }}</li>
+      <li>{{ infoBullet2 }}</li>
+      <li>{{ infoBullet3 }}</li>
+      <li>{{ infoBullet4 }}</li>
+      <li>{{ infoBullet5 }}</li>
+    </ol>
+  {% endset %}
+  {{ govukDetails({
+    summaryText: infoTakePhoto,
+    html: infoTakePhotoDetails
+  }) }}
+  <p class="govuk-body">{{ minRequirements }}</p>
+
+  <script id="uploadContent" type="application/json">
+  {
+    "isDraft": {{ isDraft }},
```

Comment:
should this be changed to 'isAwaitingApplicant2' since it's not a draft at this point?

Reply:
whoops a little copy and paste error - well spotted 👍